### ISSUE TITLE
Get it to play nice w/ Git::Repository (System::Command)

### ISSUE THREAD

**Issue by drmuey** [drmuey/p5-App-Task#9]
warning about printing to unopened file handle ORIG…, no output

When OPEN and CLOSE are set up like Tie::Handle::Base

1. everything is output from `$git->run(…)` as if you print $git->run(…);
2. `my $out = $git->run(…);` does not go into `$out` but gets printed.

**Comment by drmuey** [comment]
fiddling code:

```
    sub OPEN {
        my $self = shift;
        $self->CLOSE if defined $self->FILENO;

        # use Data::Dumper;
        # print Dumper( \@_ );

        # note open is prototyped, so the conditional is needed here:
        if (@_) { return open $self->{orig}, shift, @_ }
        else    { return open $self->{orig} }
    }   

    sub CLOSE { close shift->{orig} }
```

**Comment by drmuey** [comment]
Fixed in v0.02
